src/SignalR/clients/java/signalr/core/build.gradle

```diff
@@ -78,4 +78,4 @@ class Version {
     }
 }
 
-compileJava.dependsOn generateVersionClass
\ No newline at end of file
+compileJava.dependsOn generateVersionClass
```

Comment:
Didn't think I set VS Code to add trailing newlines _everywhere_. Will this cause problems @BrennanConroy❔

Reply:
Hope not, we'll see if the CI fails :D